src/common.py

```diff
@@ -90,7 +90,9 @@ def _load_finished(self, result):  # pylint: disable=unused-argument
 def cache_webkit(url, wait=None):
   if wait is not None:
     time.sleep(wait)
-  return Render(url).html
+  html = Render(url).html
+  print(html)
```

Comment:
In general, we should remove print statements

Reply:
I moved the print statement, with an explanatory comment about the occasional captcha, to Nevada's `fetch_data()`